vitest.config.js

```diff
@@ -0,0 +1,8 @@
+const { defineConfig } = require('vitest/config');
```

Comment:
Do we need this file if you already specified a config in `package.json`? Perhaps choose just 1 location?